### PR TITLE
Adding the correct files

### PR DESCRIPTION
**Pull request by Imad-Allal**

### CODE REVIEW

File: ask.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import os
+import openai
+from dotenv import load_dotenv
+
+load_dotenv()
+
+OPENAI_API_KEY = os.getenv("OPENAI_API_KEY")
```
Comment (achab) [inline ask.py:7]: you should add the api key to openai configuration. look at the example here: https://github.com/openai/openai-python

File: ask.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import os
+import openai
+from dotenv import load_dotenv
+
+load_dotenv()
```
Comment (achab) [inline ask.py:5]: this part works on my machine, and I'm able to get the environment variables stored in `.env` file

Comment (Imad-Allal) [reply ask.py:5]: So now we are sure that the problem is on my dotenv library, I'll see how I could fix it

File: ask.py (hunk 1 above)
Comment (Imad-Allal) [reply ask.py:7]: I have already added it, but still not working:
image

File: ask.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+from dotenv import load_dotenv
+
+
+load_dotenv()
+openai.api_key = os.getenv("OPENAI_API_KEY")
+
+def main():
+
+
+
```
Comment (achab) [inline ask.py:12]: remove those empty lines

File: ask.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+            )
+            chat = openai.ChatCompletion.create(
+                model="gpt-3.5-turbo", messages=messages
+            )
+        
+        reply = chat.choices[0].message.content
+        print(f"ChatGPT: {reply}")
+        messages.append({"role": "assistant", "content": reply})
+
+main()
```
Comment (achab) [inline ask.py:31]: add an `if` statement to run the function only if you run the file (`if __name__ ...`)

**Review by achab — APPROVED** [review]
Good changes, I've put two last comments. Feel free to merge the PR once they are addressed

**Pull request merged**